src/components/KFileUpload/KFileUpload.vue

```diff
@@ -151,7 +151,7 @@ const fileSizeErrorMessage = computed((): string => {
 
   return ''
 })
-const maximumFileSize = computed((): Number => {
```

Comment:
Yep, this was the issue; you needed `number` not `Number` 😬

Reply:
Yeah, I thought it was `evt.target?.files` but then noticed this